conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -622,7 +625,15 @@ TEST_F(
     EXPECT_NE(core_device, nullptr);
 #endif
 
-    auto processes = lzt::get_processes_state(device, count);
+    ze_result_t result{};
+    auto processes = lzt::get_processes_state(device, count, result);
+    if (result == ZE_RESULT_ERROR_UNSUPPORTED_FEATURE) {
+      LOG_INFO << "zesDeviceProcessesGetState is Unsupported!";
```

Comment:
I think we can put log as  ... Unsupported on this device/driver  for clear msg.

Reply:
Updated at line 697